src/napi/js_native_api.h

```diff
@@ -238,6 +238,8 @@ NAPI_EXTERN napi_status napi_unwrap(napi_env env, napi_value js_object,
                                     void **result);
 NAPI_EXTERN napi_status napi_remove_wrap(napi_env env, napi_value js_object,
                                          void **result);
+NAPI_EXTERN napi_status napi_create_object(napi_env env, 
```

Comment:
Did they add this symbol recently?

Reply:
It most likely already is defined.

Reply:
just moved from Zig to C++ so we can use the `NapiPrototype`, this way `napi_create_object` will work with `napi_wrap/unwrap`